app/views/audits/allocations/index.html.erb

```diff
@@ -11,6 +11,8 @@
 <div class="allocations" data-module="batch-selection">
   <%= form_tag audits_allocations_path do %>
       <%= render 'toolbar' %>
+      <%= render 'audits/common/counter' %>
```

Comment:
We should provide the number to the partial as a local variable not rely on the partial being used in a context where `content_items` is present.

Reply:
This is very true 😊